core/src/main/java/tc/oc/pgm/crafting/CraftingMatchModule.java

```diff
@@ -27,8 +29,14 @@ public CraftingMatchModule(
 
   @Override
   public void enable() {
-    for (Recipe recipe : customRecipes) {
-      match.getWorld().addRecipe(recipe);
+    if (BukkitUtils.isSportPaper()) {
+      for (Recipe recipe : customRecipes) {
+        match.getWorld().addRecipe(recipe);
+      }
+    } else {
+      for (Recipe recipe : customRecipes) {
+        Bukkit.addRecipe(recipe);
+      }
```

Comment:
Let's move this into a `WorldUtils` or `BukkitUtils` as `addRecipe(World, Recipe)`